css/layout.css

```diff
@@ -62,3 +62,13 @@
     width: 30%;
   }
 }
+
+@media screen and (min-width: 1440px) {
+  div.half-col.left {
+    margin-left: 18rem;
+  }
+
+  div.half-col.right {
+    margin-right: 15rem;
```

Comment:
Instead of doing this with margin, I would restrict the width of the container these are in, and just align the text of the second statement to the right